docs/appsStorageIssue.md

```diff
@@ -0,0 +1,57 @@
+---
+id: appsStorageIssue
+title: Apps Strorage Issue
+sidebar_label: Apps Strorage Issue
+---
+
+Here are a few possible reasons for the long "pending" of a storage order:
+1. The file is too large;
+2. The local IPFS is not working;
+3. The firewall is not turned off;
+4. Poor network conditions;
+
+To address this, you may need to:
+
+## 1. Make sure not to upload too large files
```

Comment:
I don't think it's a good idea to use "make sure". Looks like we definitely cannot do this. Maybe just putting the exact GB here is better. For example, `Recommending file size is smaller than 64G` or something similar.

Reply:
Good point